test/sagemaker_tests/mxnet/training/integration/local/test_onnx.py

```diff
@@ -34,7 +34,4 @@ def test_onnx_export(docker_image, sagemaker_local_session, local_instance_type,
                image_uri=docker_image, framework_version=framework_version,
                output_path='file://{}'.format(tmpdir))
 
-    input_path = 'file://{}'.format(os.path.join(ONNX_PATH, 'mxnet_module'))
-    mx.fit({'train': input_path})
-
-    local_mode_utils.assert_output_files_exist(str(tmpdir), 'model', ['model.onnx'])
```

Comment:
@waytrue17 what are we testing here? I see this line as the only check in this test.

Reply:
I think it tests the existence of the exported onnx file, which is redundant because we do the same at https://github.com/aws/deep-learning-containers/blob/0dc7e54bb5512f6b73a7490bbd5a4ae2db4a9413/test/sagemaker_tests/mxnet/training/resources/onnx/code/onnx_export.py#L42